examples/java/pom.xml

```diff
@@ -138,12 +137,17 @@
                   <goal>verify</goal>
                 </goals>
                 <configuration>
+                  <includes>
+                    <include>WordCountIT.java</include>
```

Comment:
Based upon the maven documentation, it is unclear whether this will work without it being prefixed by **/
Did you try this out?